### PR TITLE
feat: expose TimePicker[open], close: #5775 & #5829

### PR DESCRIPTION
**Pull request by benjycui**
close: #5775 & #5829

### CODE REVIEW

File: components/time-picker/demo/addon.md [hunk 1]
```diff
@@ -16,13 +16,30 @@ Render addon contents to timepicker panel's bottom.
...
+  handleClose = () => this.setState({ open: false })
+
+  render() {
+    return (
+      <TimePicker
+        open={this.state.open}
+        onOpen={this.toggleOpenStatus}
+        onClose={this.toggleOpenStatus}
+        addon={() => (
+          <Button size="small" type="primary" onClick={this.handleClose}>
```
Comment (benjycui) [inline components/time-picker/demo/addon.md]: Make this more React-like.

**Comment by benjycui** [comment]
pending for https://github.com/react-component/collapse/pull/67

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
ci broken

File: components/time-picker/index.en-US.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ import moment from 'moment';
 |---------------------|-----|-----|-------|
 | defaultValue        | to set default time | [moment](http://momentjs.com/) | - |
 | value               | to set time | [moment](http://momentjs.com/) | - |
+| open                | whether to popup panel | boolean | false |
+| onOpen              | a callback function which will be called while panel opening | ({ open: boolean }): void | 无 |
+| onClose             | a callback function which will be called while panel closing | ({ open: boolean }): void | 无 |
```
Comment (afc163) [inline components/time-picker/index.en-US.md]: 和 Menu 一样改成 onOpenChange 吧。

**Comment by benjycui** [comment]
Updated.

File: components/time-picker/index.en-US.md [hunk 3]
```diff
@@ -25,6 +25,8 @@ import moment from 'moment';
 |---------------------|-----|-----|-------|
 | defaultValue        | to set default time | [moment](http://momentjs.com/) | - |
 | value               | to set time | [moment](http://momentjs.com/) | - |
+| open                | whether to popup panel | boolean | false |
+| onOpenChange        | a callback function which will be called while panel opening/closing | (status: boolean): void | 无 |
```
Comment (afc163) [inline components/time-picker/index.en-US.md]: 无 => -

File: components/time-picker/index.tsx [hunk 4]
```diff
@@ -11,8 +11,10 @@ export interface TimePickerProps {
   size?: 'large' | 'default' | 'small';
   value?: moment.Moment;
   defaultValue?: moment.Moment;
+  open?: boolean;
   format?: string;
   onChange?: (time: moment.Moment, timeString: string) => void;
+  onOpenChange?: (status: boolean) => void;
```
Comment (afc163) [inline components/time-picker/index.tsx]: status => open

**Comment by benjycui** [comment]
后两个细节，我合并后再改。

**Pull request merged**